src/org/openbravo/erpCommon/ad_callouts/SE_Order_Organization.java

```diff
@@ -53,42 +58,48 @@ protected void execute(CalloutInfo info) throws ServletException {
         FieldProvider[] td = null;
 
         final String strOrgId = info.getStringParameter("inpadOrgId", IsIDFilter.instance);
-        final String strBPartnerId = info.getStringParameter("inpcBpartnerId",
-                IsIDFilter.instance);
-        final String strBPartnerLocationId = info.getStringParameter(
-                "inpcBpartnerLocationId", IsIDFilter.instance);
 
-        info.addResult("inpiscashvat",
-                CashVATUtil.isCashVAT(strinpissotrx, strOrgId, strBPartnerId,
-                        strBPartnerLocationId));
+        OBCriteria<OrgWarehouse> orgWarehouseCriteria = OBDal.getInstance().createCriteria(OrgWarehouse.class);
+        orgWarehouseCriteria.add(Restrictions.eq(OrgWarehouse.PROPERTY_ORGANIZATION, OBDal.getInstance().get(Organization.class, strOrgId)));
+        orgWarehouseCriteria.add(Restrictions.eq(OrgWarehouse.PROPERTY_ACTIVE, true));
```

Comment:
This property is not required, by default a hibernate always filters by PROPERTY_ACTIVE = true unless you tell it otherwise.